domains/Accounts/Controllers/AccountsLogoutController.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Domains\Accounts\Controllers;
+
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+
+class AccountsLogoutController extends Controller
+{
+    public function __invoke(Request $request): Response
+    {
+        $token = $request->user()->token();
+
+        $token->revoke();
```

Comment:
Nit: we can merge these lines into a single call, no?

Reply:
Done!